getting_started_scenarios/ssm_scenario/SPECIFICATION.md

```diff
@@ -0,0 +1,122 @@
+#  AWS Systems Manager Engineering Specification
+
+## Overview
+   This SDK getting started scenario demonstrates how to interact with AWS Systems Manager (SSM) using the AWS SDK. The provided code is an AWS application that demonstrates the usage of the AWS SSM SDK to interact with various AWS SSM service operations.
+
+## Scenario Program Flow
+   - Creates an SSM maintenance window with a default name or a user-provided name.
+   - Modifies the maintenance window schedule.
+   - Creates an SSM document with a default name or a user-provided name.
+   - Sends a command to a specified EC2 instance using the created SSM document and display the time when the command was invoked.
+   - Creates an SSM OpsItem with a predefined title, source, category, and severity.
+   - Updates and resolve the created OpsItem.
+   - Deletes the created SSM maintenance window and document.
+
```

Comment:
It looks like you're intending implementers to mostly follow a reference implementation, so you should mention that here and link to it.

Reply:
The intention there was to provide a high level description of the program flow.